pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -267,4 +328,4 @@ def solve_NLP_feas(solve_data, config):
         raise ValueError(
             'Problem is not feasible, check NLP solver')
```

Comment:
Now that we are modifying documentation, I would add to this ValueError: `Feasibility NLP problem is not feasible, check NLP solver output`